### DISCUSSION TITLE
Questions about the Download using the Heroic + external drive

### DISCUSSION THREAD

**Discussion by VCTGomes** [Heroic-Games-Launcher/HeroicGamesLauncher#844] [Q&A]
Hi. I'm new with the Heroic and with the macOS and I'd like to know if when I make any game download on my external drive, this file will be downloaded directly to my external drive or it'll download to internal SSD and then move to external?

I fear about the internal SSD lifespan downloading so big games that have about 100 or 200GB.

Thanks.

**Comment by flavioislima** [comment]
It will be downloaded to the place you choose directly.
